global.json

```diff
@@ -1,5 +1,7 @@
 {
-    "msbuild-sdks": {
-        "MSBuild.Sdk.Extras": "3.0.44"
-    }
+  "sdk": {
```

Comment:
Shouldn't need a global.json weird that you are

Reply:
It's due to the fact that we are building all the repo's here, without it, the API's fail to be produced as the sln build fails. Once Net 10 becomes release and there is a VS2026 Action runner then this can be removed again.